lighthouse-core/formatters/partials/table.css

```diff
@@ -51,6 +53,7 @@
   text-align: left;
   width: 250px;
   white-space: nowrap;
+  width: 440px;
```

Comment:
can we stick the increased width of a couple ranges in a few media queries? this is probably too big as eric points out when the screen is smaller